sounddevice.py

```diff
@@ -2276,9 +2276,10 @@ class PortAudioError(Exception):
     msg : str, optional
         User error message.
     code : int, optional
-        PaErrorCode value or None if there is no PaErrorCode associated
-        with the error.
-
+        PaErrorCode value.
+    hosterror_info : tuple, optional
```

Comment:
Note that I expect the tuple described here and not the object returned by GetLastHostErrorInfo